File: api/cvia/models/Submission.py
from django.db.models import Model, AutoField, ForeignKey, DateTimeField, FloatField
from .resume import Resume
from .job import Job


class Submission(Model):

    id = AutoField(primary_key=True)
    resume = ForeignKey(Resume, related_name='resume')
    job = ForeignKey(Job, related_name='job')
    created_at = DateTimeField(auto_now_add=True)
    score_1 = FloatField(null=True, default=0)
    score_2 = FloatField(null=True, default=0)
    score_3 = FloatField(null=True, default=0)
    score_4 = FloatField(null=True, default=0)
    max_score = FloatField(null=True, default=0)

File: app/app/models/submission.js
import DS from 'ember-data';

export default DS.Model.extend({
  job: DS.belongsTo('job'),
  resume: DS.belongsTo('resume'),
  score_1: DS.attr('number'),
  score_2: DS.attr('number'),
  score_3: DS.attr('number'),
  score_4: DS.attr('number'),
  max_score: DS.attr('number'),
  createdAt: DS.attr('string', {
    defaultValue() {
      return new Date();
    }
  })
});
